client/post-editor/media-modal/index.jsx

```diff
@@ -209,15 +209,15 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorClose: function() {
+	onImageEditorClose: function( event, { hasEditedImage } = { hasEditedImage: false } ) {
```

Comment:
A bit more concise, can drop the fallback empty object if second argument is always passed:

```js
{ hasEditedImage = false } = {}
```